tests/python/test_f16.py

```diff
@@ -259,12 +259,12 @@ def test_arg_f16():
     y = ti.field(dtype, shape=())
 
     @ti.kernel
-    def foo(a: ti.f16):
-        x[None] = y[None] + a
+    def foo(a: ti.f16, b: ti.f16):
```

Comment:
I suggest putting a f32 in between them to properly test the alignment of structs